index.html

```diff
@@ -0,0 +1,139 @@
+                <img id="logoimglink" src="./images/CCC_logo-removebg-preview.png">
+            </a>
+            <nav id="desktopmenu">
+                <a class="menuitem" href="https://asdt560.github.io/Capstone-HTML-CSS-Basic-JS/about.html">Home</a>
+                <a class="menuitem" href="#">Program</a>
+                <a class="menuitem" href="#">Join</a>
+                <a class="menuitem" href="#">Sponsor</a>
+                <a class="menuitem" href="#">News</a>
+            </nav>
+        </section>
+        <section id="headline">
+           <h1> <p id="welcome">"Hello, creators!"</p></h1>
+            <h2 id="title">CONTENT CREATORS <br>
+                CONVENTION <br>
+                2022</h2>
+            <p id="openingblurb">An opportunity for creators from across the world to meet and discuss their work, this year hosted in Argentina</p>
+            <p id="date">2022.10.26(WED) ~ 16(FRI)</p>
+            <p id="place">@ Buenos Aires Convention Centre</p>
+        </section>
+    </header>
+    <main>
+        <section id="program">
+            <div class="titlecontainer">
+                <h2 class="sectiontitle program">Main Program</h2>
+                <span class="titleunderline"></span>
+            </div>
+            <div id="programholder">
+                <div class="programdiv">
+                    <img class="programimg" src="./images/lecture.png">
+                    <h3 class="programtitle">Lecture</h3>
+                    <p class="programblurb">Listen toe xperienced individuals share their knowledge on a variety of subjects</p>
+                </div>
+                <div class="programdiv">
+                    <img class="programimg" src="./images/exhibition.png">
+                    <h3 class="programtitle">Exhibition</h3>
+                    <p class="programblurb">Watch aspiring novices and veterans both show sneak-peeks of their work</p>
+                </div>
+                <div class="programdiv">
+                    <img class="programimg" src="./images/forum.png">
+                    <h3 class="programtitle">Forum</h3>
+                    <p class="programblurb">Share your thoughts and discuss a variety of subjects with experts on the topic</p>
+                </div>
+                <div class="programdiv">
+                    <img class="programimg" src="./images/workshop.png">
+                    <h3 class="programtitle">Workshop</h3>
+                    <p class="programblurb">Try your hand at creating something of your own rather than just watching</p>
+                </div>
+                <div class="programdiv">
+                    <img class="programimg" src="./images/ignite.png">
+                    <h3 class="programtitle">CCC Ignite</h3>
+                    <p class="programblurb">Get opportunities to network and build connections across the world</p>
+                </div>
+            </div>
```

Comment:
As the [last code reviewer suggested](https://github.com/elad237/capstone-project/pull/1/files#r1008594662) kindly add a hover effect to these cards. :eyes: :eyes: When you put the mouse cursor over every card a white border should appear around the card. Kindly add this effect please :pray: :pray:  
![image](https://user-images.githubusercontent.com/98363075/198772711-377ea900-8712-4f2e-ae8f-8de22a2e8227.png)

Reply:
Hello @Santiago220991 please can you explain to me exactly the change you need